src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discountSevenDays = 50;
+  const discountThreeDays = 20;
+  const rentCar = 40 * days;
+  const sevenDays = 7;
+  const threeDays = 3;
+
+  if (days >= sevenDays) {
+    return rentCar - discountSevenDays;
+  }
+
+  if (days >= threeDays && days < sevenDays) {
+    return rentCar - discountThreeDays;
+  }
+
+  if (days < threeDays) {
+    return rentCar;
+  }
```

Comment:
no need in last if statement - by default should return basic cost
```suggestion
  return rentCar;
```